drivers/storage/portworx/manifest/manifest.go

```diff
@@ -96,6 +96,8 @@ type Release struct {
 	Pause                      string `yaml:"pause,omitempty"`
 	DynamicPlugin              string `yaml:"dynamicPlugin,omitempty"`
 	DynamicPluginProxy         string `yaml:"dynamicPluginProxy,omitempty"`
+	CsiLivenessProbe           string `yaml:"csiLivenessProbe,omitempty"`
+	CsiDriverWinInstaller      string `yaml:"csiDriverWinInstaller,omitempty"`
```

Comment:
Same comment as above? Is this an installer or a windows driver? If the later, let's rename it accordingly.

Reply:
this is installer, changed the name as suggested